include/taskolib/Sequence.h

```diff
@@ -69,6 +69,21 @@ namespace task {
  * // insert an action step at index 1
  * seq.insert(seq.begin() + 1, Step{Step::type_action});
  * \endcode
+ *
+ * \section Step setup script
+ *
+ * Each step can have a setup script that is executed before the intrinsic script is
+ * called (invoked with Step::set_script(const std::string&)). You can call it with
+ * set_step_setup_script(std::string).
+ * The defined variables and functions in the setup are exported in each step and you can
+ * access them in the script. Changing the previous defined variables and/or functions
+ * with a new definition from the setup is somehow isolated in the step itself and has no
+ * effect on other steps.
+ * The setup is only executed for Step types ACTION, IF, ELSEIF, or WHILE.
```

Comment:
Oh, really? In this case that is important information. So these steps are also the only ones in which we execute scripts at all?

Reply:
:+1: